### PR TITLE
Update tests with "TODO" and time_sleep

### PR DESCRIPTION
**Pull request by kvorona**

### CODE REVIEW

File: pages/start_page.py [hunk 1]
```diff
@@ -25,45 +25,48 @@ def sign_up(self, username='', email='', password=''):
...
     def email_error(self):
-        assert self.driver.find_element(by=By.XPATH,
-                                        value=".//div[contains(text(),'You must provide a valid email address.')]").is_displayed(), \
+        assert self.driver.find_element(by=By.XPATH, value=self.constants.SIGN_UP_EMAIL_ERROR).is_displayed(), \
             "We not see message 'You must provide a valid email address'"
 
     def display_sign_up_button(self):
-        assert self.driver.find_element(by=By.XPATH,
-                                        value='.//button[contains(text(),"Sign up for OurApp")]').is_displayed(), \
+        assert self.driver.find_element(by=By.XPATH, value=self.constants.SIGN_UP_BUTTON).is_displayed(), \
```
Comment (QALightLecorPython) [inline pages/start_page.py:64]: Hint: You could add some `is_displayed` method, so would not have to pass `by` each time and verification would looks better
```
assert self.is_displayed(self.constants.SIGN_UP_BUTTON)
```

File: pages/utils.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                        logger.warning(f"Catch: {err}")
+                        raise err
+                    sleep(period)
+
+        return wrapper
+
+    return decorator
+
+
+def log_wrapper(func):
```
Comment (QALightLecorPython) [inline pages/utils.py:28]: Hint: You don't have to put it in your code if you don't use it

**Review by QALightLecorPython — APPROVED** [review]
Looks good!
I have few small comments, but it's just hints

File: pages/utils.py (hunk 2 above)
Comment (kvorona) [reply pages/utils.py:28]: Thanks. I tried to add an example.

File: pages/start_page.py (hunk 1 above)
Comment (kvorona) [reply pages/start_page.py]: Thx for the hint. Update.

**Review by QALightLecorPython — APPROVED** [review]
Looks great, let's merge it!

**Pull request merged**